ui_vue/cypress.config.ts

```diff
@@ -20,6 +20,7 @@ import { defineConfig } from 'cypress';
 export default defineConfig({
   e2e: {
     specPattern: 'cypress/e2e/**/*.{cy,spec}.{js,jsx,ts,tsx}',
-    baseUrl: 'http://localhost:4173'
+    baseUrl: 'http://localhost:5173',
```

Comment:
You need to define the base url some where which is included in gitignore eg: .env